assets/pages/subscriptions-landing/subscriptionsLanding.scss

```diff
@@ -19,24 +19,29 @@
 @import '~components/readyToSupport/readyToSupport';
 @import '~components/svgs/svg';
 @import '~components/threeSubscriptions/threeSubscriptions';
-@import '~components/digitalSubscriptions/digitalSubscriptions';
-@import '~components/paperSubscriptions/paperSubscriptions';
-@import '~components/internationalSubscriptions/internationalSubscriptions';
+@import '~components/subscriptionsByCountryGroup/subscriptionsByCountryGroup';
 @import '~components/whySupport/whySupport';
 
 
 // ----- Page-Specific Styles ----- //
 
 #subscriptions-landing-page {
 
-  .component-digital-subscriptions, .component-international-subscriptions {
+  .component-subscriptions-by-country-group {
     @include multiline-top-border;
 
-    border-top: none;
+    .component-three-subscriptions:nth-of-type(1) {
+      border-top: none;
+    }
   }
 
   .component-cta-link--android {
     margin-top: $gu-v-spacing;
   }
 
+  // As a test we are tweaking the layout of the US product bundles
```

Comment:
We can probably remove this comment now as this is the new default for all the international pages. In fact I think this CSS rule is probably redundant as well

Reply:
Which are we settling on as the default? The US currently has `font-size: 24px`, everywhere else has `font-size: 30px`.